components/library/tntnet/Solaris/http-tntnet.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0"?>
+<!--
+CDDL HEADER START
+
+This file and its contents are supplied under the terms of the
+Common Development and Distribution License ("CDDL"), version 1.0.
+You may only use this file in accordance with the terms of version
+1.0 of the CDDL.
+
+A full copy of the text of the CDDL should have accompanied this
+source.  A copy of the CDDL is also available via the Internet at
+http://www.illumos.org/license/CDDL.
+
+CDDL HEADER END
+-->
+
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+    Copyright (c) 2007, 2011, Oracle and/or its affiliates. All rights reserved.
+    Copyright 2011 EveryCity Ltd. All rights reserved.
+    Copyright (c) 2016 Jim Klimov
+-->
+
+<service_bundle type='manifest' name='tntnet'>
+
+<service
+	name='network/http/tntnet'
+	type='service'
+	version='1'>
+
+	<!--
+	  Because we may have multiple instances of network/http
+	  provided by different implementations, we keep dependencies
+	  and methods within the instance.
+	  Also, in case of tntnet in particular, it is likely to have
+	  different webapps served by different service instances with
+	  their customized configuration files, access rights, etc.
+	-->
+
+	<instance name='default' enabled='false'>
+		<!--
+		  Wait for network interfaces to be initialized.
+		-->
+		<dependency name='network'
+		    grouping='require_all'
+		    restart_on='error'
+		    type='service'>
+		    <service_fmri value='svc:/milestone/network:default'/>
+		</dependency>
+
+		<!--
+		  Wait for all local filesystems to be mounted.
+		-->
+		<dependency name='filesystem-local'
+		    grouping='require_all'
+		    restart_on='none'
+		    type='service'>
+		    <service_fmri
+			value='svc:/system/filesystem/local:default'/>
+		</dependency>
+
+		<!--
+		  Wait for automounting to be available, as we may be
+		  serving data from home directories or other remote
+		  filesystems.
+		-->
+		<dependency name='autofs'
+		    grouping='optional_all'
+		    restart_on='error'
+		    type='service'>
+		    <service_fmri
+			value='svc:/system/filesystem/autofs:default'/>
+		</dependency>
+
+		<!--
+		  Wait for database to be available if enabled, as this is
+		  a web-appplication server. They usually need databases ;)
+		  NOTE: In actual deployment, you'd likely need to tune this.
+		-->
+		<dependency name='database'
```

Comment:
Maybe. Or we could list all postgreses and perconas though;)

--
Typos courtesy of GMail on my Samsung Android
24.03.2016 16:07 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/tntnet/Solaris/http-tntnet.xml
> <https://github.com/OpenIndiana/oi-userland/pull/1777#discussion_r57331089>
> :
>
> > +		  filesystems.
> > +		-->
> > +		<dependency name='autofs'
> > +		    grouping='optional_all'
> > +		    restart_on='error'
> > +		    type='service'>
> > +		    <service_fmri
> > +			value='svc:/system/filesystem/autofs:default'/>
> > +		</dependency>
> > +
> > +		<!--
> > +		  Wait for database to be available if enabled, as this is
> > +		  a web-appplication server. They usually need databases ;)
> > +		  NOTE: In actual deployment, you'd likely need to tune this.
> > +		-->
> > +		<dependency name='database'
>
> Perhaps, leave it commented? We can't predict what dbms will user use.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1777/files/f3f31f4d279e4b02b87ceb5fb92b8a68c26a139a..cc739bc398ec10c740b0d3937f6723360fc64c71#r57331089>
>